backend/src/types.ts

```diff
@@ -115,9 +115,9 @@ export type BuildKind = {
     output: {
       to: {
         name: string;
-      }
-    }
-  }
+      };
```

Comment:
Heh, I put the same in my PR, if your PR is merged first I think ill need to rebase

Reply:
Doesn't matter -- if you both make the same change git diff engine will make it a moot changeset to your PR.